dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -1001,6 +996,53 @@ public String formatObj(Object objs) {
         return ans;
     }
 
+    private String formatObject(Object objs, int depth) {
+        if (objs == null) {
+            return null;
```

Comment:
This should return `nullParamString` not `null`

Reply:
done